internal/pkg/fixtures/sharedcode.go

```diff
@@ -24,7 +24,6 @@ func CreateSharedCode(t *testing.T, state model.ObjectStates, naming *naming.Reg
 		Remote: &model.Branch{BranchKey: branchKey},
 	}
 	assert.NoError(t, state.Set(branchState))
-	assert.NoError(t, naming.Attach(branchState.Key(), branchState.PathInProject))
```

Comment:
V tomto PR riesim problem, na ktory si narazil uz aj ty.
Ked sa prida novy objekt (branch/config/row) do `state.Registry`, tak sa musi pridat este do `naming.Registry`).

To teraz uz odpadava. `naming.Registry` je sucastov `state.Registry`.
V dalsich PR to spravi jednoduchsim aj kod na dalsich miestach.